src/glue/jobs/json_to_parquet.py

```diff
@@ -253,13 +338,16 @@ def add_index_to_table(
     Returns:
         awsglue.DynamicFrame with index columns
     """
+    logger = glue_context.get_logger()
     _, table_data_type = table_name.split("_")
     this_table = unprocessed_tables[table_key].toDF()
+    logger.info(f"Adding index to {table_name}")
     if table_key == table_name: # top-level fields already include index
         for c in list(this_table.columns):
```

Comment:
Nit: I'm not a huge fan of single letter variable because they aren't as descriptive, but here, I understand `c` is a `column`